src/zenml/cli/server.py

```diff
@@ -336,6 +336,11 @@ def deploy(
     with track_handler(
         event=AnalyticsEvent.ZENML_SERVER_DEPLOYED
     ) as analytics_handler:
+        try:
+            terraform_utils.verify_terraform_installation()
+        except RuntimeError as e:
+            cli_utils.error(str(e))
```

Comment:
Can this not be a bit nicer? If it raises this error we know they didn't install the pip package, right, so can't we just raise a clean error?

Reply:
That's exactly what `verify_terraform_installation` in the try-block does